solutions/biggestValue.js

```diff
@@ -0,0 +1,10 @@
+const biggestValue=(obj,objKeys=Object.keys(obj),i=0,maxKey=objKeys[i])=>{
+  if(i==objKeys.length){
+    return maxKey;
```

Comment:
I think this should be named as maxVal. You are storing values, not keys.

Reply:
done! thanks